submarine-security/spark-security/pom.xml

```diff
@@ -547,10 +572,25 @@
       <id>spark-2.4</id>
       <properties>
         <spark.version>2.4.5</spark.version>
+        <guava.version>15.0</guava.version>
         <scalatest.version>3.0.3</scalatest.version>
       </properties>
     </profile>
 
+    <profile>
+      <id>spark-3.0</id>
+      <properties>
+        <spark.version>3.0.1</spark.version>
+        <scala.version>2.12.12</scala.version>
```

Comment:
Spark is compiled w/ scala 2.12.10, can we make it consistent w/ spark?

Reply:
sure, i well be change to `scala 2.12.10`